### DISCUSSION TITLE
Are you comfortable approving each small pull request?

### DISCUSSION THREAD

**Discussion by Protiguous** [AdamWhiteHat/BigDecimal#31] [Discussions]
@AdamWhiteHat, are you more comfortable approving each small pull request?

Or are you okay with allowing me to pull fully-tested incremental updates directly into main (master) to save time on my end? (Not having to wait for the next push/pull/approve cycle.)

I tend to program in bursts while I have time.

**Comment by Protiguous** [comment]
@AdamWhiteHat I have not heard back from you about reviewing pull requests into `main`/`master`.

Will you create an `dev` type of branch that we can PR into, and then you can PR what you want into `main`/`master`.

**Reply by AdamWhiteHat** [reply]
I created a development branch.

**Reply by Protiguous** [reply]
Excellent! Thank you.
